das/daser.go

```diff
@@ -138,8 +141,11 @@ func (d *DASer) sample(ctx context.Context, sub header.Subscription, checkpoint
 
 		startTime := time.Now()
 
-		err = d.da.SharesAvailable(ctx, h.DAH)
+		ctxWithSpan, span := otel.Tracer(tracerName).Start(ctx, "sample")
```

Comment:
AFAIK, the `otel.Tracer` should be defined once globally and then accessed multiple times from the code.

Reply:
The guide also mentions this https://opentelemetry.uptrace.dev/guide/go-tracing.html#quickstart